web/src/settings_users.ts

```diff
@@ -252,24 +264,36 @@ function populate_users(): void {
                 presence_data_fetched = true;
                 active_section.create_table(active_user_ids);
                 deactivated_section.create_table(deactivated_user_ids);
-                create_role_filter_dropdown($("#admin-user-list"), active_section, active_user_ids);
-                create_role_filter_dropdown(
+                active_users_role_dropdown = create_role_filter_dropdown(
+                    $("#admin-user-list"),
+                    active_section,
+                    get_roles_count_for_active_users,
+                );
+                deactivated_users_role_dropdown = create_role_filter_dropdown(
```

Comment:
Maybe this code for creating tables and dropdowns used here and below, can be deduplicated by having a new function.